blob-parser/readme.md

```diff
@@ -1,11 +1,14 @@
-# BlobTrigger - Python
+# Blob Parser
 
-The `BlobTrigger` makes it incredibly easy to react to new Blobs inside of Azure Blob Storage. This sample demonstrates a simple use case of processing data from a given Blob using Python.
+Blob parser parses the HTML from the Blob Storage into JSON for the cosmos DB.
+It then inserts the data into the DB.
 
-## How it works
+Note that this function produces a lot of output after starting the function app, because it is continuously checking for its trigger. For this reason, you may want to disable this function during local development if it's not needed. To do this, click the 'A' in VS Code sidebar, find the function at the bottom under Workspace > Local Project > Functions, right-click on it and click Disable. You can use the same menu to re-enable it when you need it again.

```

Comment:
Do we mention installing the Azure extension for VS Code? If not, this might be a good place to mention it.

Reply:
Good catch. Added